### PR TITLE
Added dummycheck

### PR DESCRIPTION
**Pull request by ev1stensberg**
Swapped around so we don't need to use `componentName = componentName || something`

Does basically the same thing ( I think??  ) but does a conditional instead so we can cross out invoking the object iteself to check if it's an object or string.

Feedback would be great. Passed all tests, but please, do check if returning the string is proper! 

❤️

### CODE REVIEW

**Comment by sampeka** [comment]
The former is much more expressive than the latter, and the premise of this being optimised is a false one. Why test for null and undefined explicitly when null and undefined are falsey?

File: modules/RouteUtils.js [hunk 1]
```diff
@@ -10,7 +11,15 @@ export function isReactChildren(object) {
 }
 
 function checkPropTypes(componentName, propTypes, props) {
-  componentName = componentName || 'UnknownComponent'
+
+  componentName = {} || ''
```
Comment (ryanflorence) [inline modules/RouteUtils.js]: this will always be `{}`, what's the goal with this PR?

**Comment by ev1stensberg** [comment]
Thanks for the notification, swapping it around as we speak :smile:

**Comment by ev1stensberg** [comment]
Closing this, fixing and let's resume this in the new PR, will go in detail there.

**Pull request closed without merging**